packages/perseus/src/widgets/interactive-graphs/grid.tsx

```diff
@@ -26,8 +28,11 @@ const axisOptions = (
 
 export const Grid = (props: GridProps) =>
     props.markings === "none" ? null : (
-        <Coordinates.Cartesian
-            xAxis={axisOptions(props, 0)}
-            yAxis={axisOptions(props, 1)}
-        />
+        <>
+            <Coordinates.Cartesian
+                xAxis={axisOptions(props, 0)}
+                yAxis={axisOptions(props, 1)}
+            />
+            <GridArrows />
```

Comment:
I'm not sure `props.markings === "none"` is the right condition to check for this. There are three possible settings for `markings`: "graph", "grid", and "none", I believe. I think we only render the axes when `markings` is "graph".